cmd/account/main.go

```diff
@@ -8,6 +8,8 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/dcrodman/archon/internal/debug"
+
 	"github.com/dcrodman/archon"
 	"github.com/spf13/viper"
```

Comment:
Huh, the linter should've complained about this too. The structure it wants is:
```
// golang stdlib imports

// third party imports

// local imports
```

Reply:
I've actually run goimports on this file. Dunno why works like this here

Reply:
This was probably added before I had VSCode running `goimports`. As far as I know, goimports won't change existing ordering. So it's probably my fault